charts/influxdb/templates/backup-cronjob.yaml

```diff
@@ -70,7 +74,7 @@ spec:
           containers:
           {{- if .Values.backup.gcs }}
```

Comment:
These conditions are newly truethy, since the elements available in default `Values.yaml` (also for azure and s3(, don't they?

Reply:
Reverted [71cf288](https://github.com/influxdata/helm-charts/pull/462/commits/71cf2882310c316fdab5c2515a9243c6bf2ddc1d) as it breaks/complicates things.